### PR TITLE
Mobile menu creation

### PR DESCRIPTION
**Pull request by massaillon**
We have created the mobile menu as per the below requirements:

    - When the user clicks (or taps) the hamburger button, the mobile menu appears.
    - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
    - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
    - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,21 +15,25 @@
...
+                <a href="#form" class="nav-link" >Contact</a>
             </li>
         </ul>
-        <i class="fa-solid fa-bars fa-xl"></i>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            
+        </div>
```
Comment (RolandM99) [inline index.html]: - [ ] The hamburger menu should have the same color as on the original design. Please, kindly make sure that your menu bar(hamburger) has the same color as the color of your website logo.
image

File: style.css [hunk 2]
```diff
@@ -46,18 +46,68 @@ body {
...
+  left: 100%;
+  top: 70px;
+  flex-direction: column;
+  background-color: #6070ff;
+  width: 100%;
+  text-align: start;
+  transition: 0.3s;
+  min-height: 700px;
+  z-index: 10;
+  opacity: 95%;
```
Comment (RolandM99) [inline style.css]: - [ ] Good job so far by implementing the mobile menu, now it's functional but, to make it accepted by the standard and also presentable you have to fit exactly the design provide by the original design from Figma.

| Your design | Figma design |
| --------------| --------------- |
| image| image|

- Kindly, make sure that you fit exactly this template provided by the Figma design. background-color, font, size, and color should be the same. Let's rock, fix this issue 😃

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @massaillon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index copy.html [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+        <ul class="nav-menu hide">
+            <li class="nav-item">
+                <a  href="#main-home" class="nav-link">Portfolio</a>
+            </li>
+            <li class="nav-item">
+                <a href="#About-Me" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="#form" class="nav-link" >Contact</a>
+            </li>
```
Comment (roshan-bajgain) [inline index copy.html]: - [ ] Kindly adjust the spacing as there is more spacing from the top.
- [ ] If you see the background blur then It is not like the Figma design so Kindly make the less blur to stick with Figma design.
### Figma design
spac

### Your design

adjust

File: index copy.html [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+        </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            
+        </div>
```
Comment (roshan-bajgain) [inline index copy.html]: - [ ] If you see the Figma design then the `X` buttons are not similar to the design kindly make the design similar with the Figma 
### your design
his
### Figma design

figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @massaillon, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -11,25 +11,27 @@
...
-
-    <nav>
-        <ul>
-            <li>
-                <a href="#">Portfolio</a>
+    <nav class="navbar">
+        <a id="logo-container" href="#">Welcome</a>
+        <ul class="nav-menu hide">
+            <li class="nav-item">
+                <a  href="#main-home" class="nav-link">Portfolio</a>
```
Comment (thecodechaser) [inline index.html:18]: - [ ] You did a good job on the functionalities but there is a little issue, When I click on any of the mobile-menu options, It takes to contact form. Please fix this issue and make sure when users click on the mobile-menu options it takes to the correct part of the webpage.

File: index.html [hunk 6]
```diff
@@ -11,25 +11,27 @@
...
-        <a href="#">Welcome</a>
-    </div>
-
-    <nav>
-        <ul>
-            <li>
-                <a href="#">Portfolio</a>
+    <nav class="navbar">
+        <a id="logo-container" href="#">Welcome</a>
+        <ul class="nav-menu hide">
```
Comment (thecodechaser) [inline index.html:16]: - [ ] Please apply the blur effect to the mobile menu background and make sure it matches with [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design.
  - For blur effect you can use this CSS `backdrop-filter: blur(5px);` change px according to your design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @massaillon👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] This is a pair-programming project, every member of your group needs to contribute to this project. I am requesting some changes Please make sure to implement the changes with your group. We will see the commit history.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @massaillon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**